lint.whitelist

```diff
@@ -398,31 +398,22 @@ NON-EXISTENT-REF: css/CSS2/normal-flow/blocks-017.xht
 ## Whitespace rules that we can't enforce yet
 INDENT TABS: css/compositing/*
 INDENT TABS: css/CSS2/*
-INDENT TABS: css/css-align/*
 INDENT TABS: css/css-backgrounds/*
 INDENT TABS: css/css-color/*
 INDENT TABS: css/css-conditional/*
-INDENT TABS: css/css-counter-styles/*
 INDENT TABS: css/css-flexbox/*
 INDENT TABS: css/css-fonts/*
-INDENT TABS: css/css-grid/*
-INDENT TABS: css/css-images/*
-INDENT TABS: css/css-lists/*
+INDENT TABS: css/css-images/support/1x1-green.svg
```

Comment:
was this addition intended?

Reply:
Yes, that file is weird ("Generator: Adobe Illustrator", `transform="matrix(1 0 0 1.375 0.0015 -0.313)" `, plenty of unneeded markup) and I didn't want to deal with it.